### ISSUE TITLE
Exiting AR session in WebXR Viewer leads to white background

### ISSUE THREAD

**Issue by hybridherbst** [needle-tools/needle-engine-support#115]
### Describe the bug 💬

Reproduces on: 
https://engine.needle.tools/samples/ar-overlay/

only in WebXR Viewer (works fine on Android WebXR).

When entering an AR session and then leaving it again, the browser turns white and no elements are displayed.

### Operating System 👩‍💻

Windows

### What browsers are you seeing the problem on? 🏄‍♂️

Mozilla XR

### Editor Version 🎲

2020.3

### Needle Exporter version

2.47.2-pre

### Steps to reproduce 🔢

```bash
1. Visit https://engine.needle.tools/samples/ar-overlay/ on Mozilla WebXR Viewer on iOS
2. Enter AR
3. Leave AR
4. Note browser turns white and nothing renders anymore
```

### Reproduction ♻

https://engine.needle.tools/samples/ar-overlay/

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Vite issue and not a Needle-Engine-specific issue. For example, if it's a Unity related bug, it should likely be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [ ] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hybridherbst** [comment]
So they're indeed setting all root childs to `display:none;`, but a few lines below are resetting the state on session end. Seems that part doesn't fully work for some reason.
https://gist.github.com/hybridherbst/f55fbf25f05ce2bb93e5a9ea351ea2d2#file-webxr-2-0-js-L5166-L5176

Note: Maybe we should embed that file directly instead of loading it from some external server that may go away...

**Comment by marwie** [comment]
should be fixed in next version > 2.48.x-pre
